books.js

```diff
@@ -1,62 +1,66 @@
 /* eslint-disable linebreak-style */
-/* eslint-disable no-use-before-define */
-/* eslint-disable no-unused-vars */
-const addBtn = document.getElementsByClassName('add-btn')[0];
-const bookSection = document.getElementsByClassName('books')[0];
-let books = [];
+/* eslint-disable max-classes-per-file */
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
 
```

Comment:
- [ ] I saw that you are not using ESLint in your linters setup. ESLint helps you to identify errors in your JavaScript files. These errors can break your whole application :fearful: Kindly use ESLint in your linters setup :+1: